tests/test_wsgi.py

```diff
@@ -49,7 +60,31 @@ def test_route_with_upload_service_absent(self, mocker):
         url = '/'
 
         response = client.get(url)
-        assert response.get_data() == \
-            b'{"message":"upload-service not operational",' \
-            b'"status":"Error","version":"0.0.1"}\n'
+
+        output = {
+            "message": "upload-service not operational",
+            "status": "Error",
+            "version": VERSION
+        }
+        assert json.loads(response.get_data()) == output
         assert response.status_code == 500
+
+
+class TestGetPublish:
```

Comment:
don't quite see we need a separate class for this test. Probably don't even need any classes in this suite.

Reply:
2 options -

We could
- either remove the class for this test, and just have a function
- or rename the class to `TestPublish` ( and add the test for POST publish separately)

How about we do the second?

Reply:
So the second will have us only 1 class in this file, right?

Reply:
yes, one class for testing the Publish route for GET and POST

Reply:
nevermind, I have changed that to be a function